curriculum/challenges/english/25-front-end-development/lab-travel-agency-page/669e2f60e83c011754f711f9.md

```diff
@@ -48,13 +49,21 @@ assert.match(
 );
 ```
 
-You should have a `meta` element within your `head` element.
+You should have two `meta` elements within your `head` element.
 
 ```js
-assert.match(code, /<\s*head\s*>[\s\S]*<\s*meta[\s\S]*>[\s\S]*<\/\s*head\s*>/i);
+assert.match(code, /<\s*head\s*>[\s\S]*(<\s*meta[\s\S]*>[\s\S]*){2,}<\/\s*head\s*>/i);
```

Comment:
this will not work if the `title` or other element is between the two `meta`, right?